src/Network/Probe.php

```diff
@@ -1716,6 +1716,8 @@ private static function pumpio($webfinger, $addr)
 	 */
 	private static function twitter($uri)
 	{
+		self::$baseurl = '';
```

Comment:
I think that we should reset that value some lines later - after the address check.

Reply:
Fair.